jekyll/_cci2/persist-data.md

```diff
@@ -1,79 +1,92 @@
+## Workspaces
+{: #workspaces }
 
-Workspaces are not shared between pipeline runs. The only time a workspace can be accessed after the pipeline has run is when a workflow is rerun within the 15
-day limit.
+Workspaces are used to transfer data to downstream jobs as the workflow progresses. When a workspace is declared in a job, files and directories can be added to it. Each addition creates a new layer in the workspace filesystem. Downstream jobs can then use this workspace for their own needs, or add more layers on top.
 
-For more information on using workspaces to persist data throughout a workflow, see the [Workflows]({{site.baseurl}}/2.0/workflows/#using-workspaces-to-share-data-among-jobs) guide. Also see the [Deep Diving into CircleCI Workspaces](https://circleci.com/blog/deep-diving-into-circleci-workspaces/) blog post.
+### Workspace optimization
+{: #workspace-optimization }
 
-## Using artifacts
-{: #using-artifacts }
+If you notice your workspace usage is high and would like to reduce it, try searching for the `persist_to_workspace` command in your `.circleci/config.yml` file to find all jobs utilizing workspaces and determine if all items in the path are necessary.
 
-![artifacts data flow]( {{ site.baseurl}}/assets/img/docs/Diagram-v3-Artifact.png)
+* For more information on workspace optimization, configuration, and expiration, see the [Using Workspaces]({{site.baseurl}}/2.0/workspaces/) page.
+* For more information on workflows, see the [Workflows]({{site.baseurl}}/2.0/workflows/) page. 
+* Also see the [Deep Diving into CircleCI Workspaces](https://circleci.com/blog/deep-diving-into-circleci-workspaces/) blog post.
 
-**Artifacts are stored for up to 30 days.**
+## Artifacts
+{: #artifacts }
 
 Artifacts are used for longer-term storage of the outputs of your pipelines. For example if you have a Java project, your build will most likely produce a `.jar` file of your code. This code will be validated by your tests. If the whole build/test process passes, then the output of the process (the `.jar`) can be stored as an artifact. The `.jar` file is available to download from our artifacts system long after the workflow that created it has finished.
 
 If your project needs to be packaged, say an Android app where the `.apk` file is uploaded to Google Play, you would likely wish to store it as an artifact. Many users take their artifacts and upload them to a company-wide storage location such as Amazon S3 or Artifactory.
 
-For more information on using artifacts to persist data once a job has completed, see the [Storing Build Artifacts]({{site.baseurl}}/2.0/artifacts/)
-guide.
+### Artifact optimization
+{: #artifact-optimization }
 
-## Managing network and storage use
-{: #managing-network-and-storage-use }
+Optimization options will be different for each project depending on what you are trying to accomplish. You can try the following actions to reduce network and storage usage:
```

Comment:
I was going to say just say storage usage here rather than network and storage because artifacts aren't subject to billing, but we are talking about speed optimisaiton too so still valid I think.

Reply:
I also think we can be vague since it's entirely possible we will charge for artifacts in the future.